src/actions/motion.ts

```diff
@@ -162,6 +162,9 @@ abstract class MoveByScreenLine extends BaseMovement {
     });
 
     if (vimState.currentMode === ModeName.Normal) {
+      if (this.keys[0] === 'g' && this.keys[1] === '$') {

```

Comment:
Is there any value in toggling based on the `movementType` rather than the keys themselves? I'm thinking of a potential situation where someone remaps `g$` to something else.

``` TypeScript
      if (this.movementType === 'wrappedLineEnd') {   
```

Reply:
That's what I tried first, and it led to some problems with other tests. Perhaps I could check for keys and movement type?

Reply:
Are you sure using `movementType` causes issues with tests, or was that simply a flaky test failing? I just tried running the tests with these changes and had two successful runs.